pages/instructorRequestCourse/instructorRequestCourse.ejs

```diff
@@ -0,0 +1,91 @@
+                                <tr>
+                                    <th>Short name</th>
+                                    <th>Title</th>
+                                    <th>Status</th>
+                                    <th>Details</th>
+                            </thead>
+                            <tbody>
+                                <% course_requests.forEach(function(req, i) { %>
+                                    <tr>
+                                        <td><%= req.short_name %></td>
+                                        <td><%= req.title %></td>
+                                        <td><%- include('approvalStatusIcon', {status: req.status}); %> </td>
+                                        <td><%= req.details %></td>
+                                    </tr>
+                                <% }); %>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            <% } /* non-zero course requests */ %>
+
+            <!-- New Request -->
+            <div class="card mb-4">
+                <div class="card-header bg-primary text-white d-flex align-items-center">
+                    Request a New Course
+                </div>
+                <form class="question-form" name="course-request" method="POST">
+                    <div class="card-body">
+                        <% if (!has_one_course) { %>
+                            <!-- Small blurb for people that have never requested a course before -->
+                            <p>
+                                To request a course space, you need to provide the <strong>course rubric and number</strong> (for example, XC 101) and the course's <strong>title</strong>. You may also optionally provide your GitHub username if you would like the ability to edit content offline.
+                            </p>
+                        <% } else { %>
+                            <p>
+                                Your course request is pending. Once your request is reviewed by a member of staff and your course space is created, you will receive an email with further instructions and resources to get started.
+                            </p>
+                        <% } %>
+
+                        <div class="form-group">
+                            <label for="cr-shortname">Course Rubric and Number</label>
+                            <input type="text" class="form-control" name="cr-shortname" placeholder="XC 101" pattern="[a-zA-Z]+ [a-zA-Z0-9]+" title="this is a series of letters, followed by a space, followed by a series of numbers and/or letters" required>
+                        </div>
+                        <div class="form-group">
+                            <label for="cr-title">Course Title</label>
+                            <input type="text" class="form-control" name="cr-title" placeholder="Example Course" minlength="1" required>
```

Comment:
Again, I suggest using a placeholder that is real, so that it's clear how to replace it. For example, "Introductory College Algebra" or something like that.